### DISCUSSION TITLE
关于获取多个收藏动画的最新一集

### DISCUSSION THREAD

**Discussion by KaeCoKio** [bangumi/server#220] [Q&A] [answered]
想获取多个收藏的动画当前最新更新到哪一集，萌新只能想到调用户收藏接口拿到多个subject_id后再用subject_id一个一个调章节信息接口，再用episode的airdate比对当前日期判断哪个是最新一集。。。这样客户端发的请求有点多运行起来应该会比较慢，请问下大佬们有什么好的方法😢

**Comment by trim21** [comment, marked as answer]
暂时没啥好办法，没做batch API。

**Reply by chivehao** [reply]
这个好像也可以通过webhook实现，

对应有需要的用户配置自己的webhook，

当剧集更新的时候，给对应用户的webhook发个通知，带上剧集id或者条目id和sort。
